deployment/docker/REQUIREMENTS.txt

```diff
@@ -23,7 +23,7 @@ setuptools==18.5
 WeasyPrint
 requests<2.12
 beautifulsoup4
-django-allauth
+django-allauth==0.33.0
```

Comment:
I would prefer us to specify this as: `django-allouth>=0.33,<0.34` minor point releases are made to fix critical bugs like security as well as just overall niceness...

Reply:
Thanks @cchristelis 
I updated.